src/class_sqlThread.py

```diff
@@ -392,6 +392,22 @@ def run(self):  # pylint: disable=too-many-locals, too-many-branches, too-many-s
                 ' and removing the hash field.')
             self.cur.execute('''update settings set value=10 WHERE key='version';''')
 
+        # Add a new table: chat and chatdata for storing chating conversation
+        item = '''SELECT value FROM settings WHERE key='version';'''
+        parameters = ''
+        self.cur.execute(item, parameters)
+        currentVersion = int(self.cur.fetchall()[0][0])
+        if currentVersion == 10:
+            self.cur.execute('''DROP TABLE chat''')
```

Comment:
DROP TABLE IF EXISTS

Reply:
yes I have already removed it in next push